crates/hsdev/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "hsdev"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+description = "A simple diesel postgres migrator that uses TOML files"
+repository = "https://github.com/juspay/hyperswitch.git"
+readme = "docs/README.md"
```

Comment:
Could you update this to pick the `edition`, `license` and `rust-version` from the Cargo workspace?

Something like this:
https://github.com/juspay/hyperswitch/blob/3d53fd018a2b14465bf3cc1557a483e98da07f9b/crates/masking/Cargo.toml#L5-L8

And you can move the `README` from `crates/hsdev/docs/README.md` to `crates/hsdev/README.md`?